autokeras/pretrained/text_classifier.py

```diff
@@ -0,0 +1,139 @@
+import numpy as np
+import os
+import tempfile
+import torch
+
+
+from autokeras.pretrained.base import Pretrained
+from autokeras.text.pretrained_bert.tokenization import BertTokenizer
+from autokeras.text.pretrained_bert.modeling import BertForSequenceClassification
+from autokeras.utils import download_file_from_google_drive, get_device
+from torch.utils.data import TensorDataset, DataLoader, SequentialSampler
+
+
+SENTIMENT_ANALYSIS_MODEL_ID = '15kIuZrzWdoEpmZ842ufZHm3B3QZFpfLu'
+TOPIC_CLASSIFIER_MODEL_ID = '1U3O9wffh-DQ7BDIezKYWcDYkM9Cly8Yb'
```

Comment:
Would you move these two to constant.py?
Follow the style of ObjectDetector and FaceDetector.